modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/list/List.es.js

```diff
@@ -30,7 +38,7 @@ export default ({data, field, totalEntries, type}) => {
 						<li key={index}>
 							{type == 'color' ? (
 								<Color hexColor={field} />
-							) : (
+							) : type == 'date' ? formatDate(field) : (
```

Comment:
@alinedoleron, please avoid use nested ternary operators, it affects code legibility. 😉